tests/hook-components/hook-component-use-bug.test.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react"
 import test from "ava"
-import { useBug } from "src/component-hooks"
+import { useChip } from "src/component-hooks"
 import { getTestFixture } from "tests/fixtures/get-test-fixture"
 import { su } from "@tscircuit/soup-util"
 
-test("useBug", async (t) => {
+test("useChip", async (t) => {
   const { render } = getTestFixture(t)
-  const U1 = useBug("U1", {
+  const U1 = useChip("U1", {
```

Comment:
You should copy this test to ensure things are still working for backwards compat